cmd/pj-rehearse/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/openshift/ci-operator-prowgen/pkg/diffs"
+)
+
+type options struct {
+	configPath     string
+	jobConfigPath  string
+	decorationPath string
+}
+
+func gatherOptions() options {
+	o := options{}
+	flag.StringVar(&o.configPath, "config-path", "/etc/config/config.yaml", "Path to Prow config.yaml")
+	flag.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
+	flag.StringVar(&o.decorationPath, "decoration-path", "./", "Path where the repository has been cloned.")
```

Comment:
I am not that happy with the `decorationPath` name, as it does not really describe what it is. I think we should accept symmetrical options for master and under-test configs. Either accept paths to full `openshift/release` repo clones (one for master, one for PR) and compute the `jobConfig` paths by knowing the layout of `openshift/release`, or accept two sets of options (again, one for master, one for pr) pointing to needed directories directly.